static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js

```diff
@@ -14,21 +14,14 @@ define([
                     trackSamePageLinkClick: sinon.spy(),
                     trackExternalLinkClick: sinon.spy()
                 })
-                .mock('common/modules/analytics/omniture', {
-                    go: sinon.spy(),
-                    trackSamePageLinkClick: sinon.spy(),
-                    trackExternalLinkClick: sinon.spy()
```

Comment:
Sorry Dom, I don't follow. With this PR, will the page still wait to register clicks to external websites so we don't lose that data?

I remember chatting about this with @gtrufitt a couple weeks ago. Then it was something that would have to be implemented before Omniture is taken out. It is pretty important data to capture and I just want to make sure we aren't going to lose it.